2-what-is-a-mutation-observer-and-when-to-use-it/README.md

```diff
@@ -0,0 +1,21 @@
+# What is a Mutation Observer and when should you use it?
+
+## What is is
+Explain what it does - listens for any change in an element's contents
+Explain when it came into being - link to the [mozilla doc](https://developer.mozilla.org/en-US/docs/Web/API/MutationObserver)
+
+## When to use it
```

Comment:
When might be more useful above the what since it gives a frame of reference and makes it easier to understand the what

Reply:
love it